migrator/migrations/m_175_to_m_176_create_notification_schedule_table/migration_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build sql_integration
+
+package m175tom176
+
+import (
+	"context"
+	"testing"
+
+	notificationschedulestore "github.com/stackrox/rox/migrator/migrations/m_175_to_m_176_create_notification_schedule_table/notificationschedulestore"
+	pghelper "github.com/stackrox/rox/migrator/migrations/postgreshelper"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMigration(t *testing.T) {
+	ctx := sac.WithAllAccess(context.Background())
+	pgTest := pghelper.ForT(t, false)
+	assert.NotNil(t, pgTest)
+
+	storage := notificationschedulestore.New(pgTest.DB)
+	// Test get from the not-created-table returns an error
+	_, _, errPre := storage.Get(ctx)
+	assert.ErrorContains(t, errPre, "relation \"notification_schedules\" does not exist")
+
+	// Create the table
+	assert.NoError(t, createNotificationScheduleTable(pgTest.DB, pgTest.GetGormDB()))
+
+	// Test get from the created table returns nil
```

Comment:
Let's cover also that you can a) upsert an entry and b) also retrieve said entry.

Reply:
c) delete an entry d) retrieve post-delete does not return any data